src/components/Reports/SeedbankSection.tsx

```diff
@@ -148,18 +134,24 @@ export default function SeedbankSection(props: SeedbankSectionProps): JSX.Elemen
   );
 }
 
+const useStyles = makeStyles((theme: Theme) => ({
+  infoCardStyle: {
+    padding: 0,
+  },
+}));
+
 type InfoFieldProps = {
   id: string;
   label: string;
   editable: boolean;
   value: string | number;
   onChange: (value: any) => void;
-  className: string;
   type: 'text' | 'date';
 };
 
 function InfoField(props: InfoFieldProps): JSX.Element {
-  const { id, label, editable, value, onChange, className, type } = props;
+  const { id, label, editable, value, onChange, type } = props;
+  const classes = useStyles();
```

Comment:
This class didn't have to be passed in through props. Use `useStyles` directly in this component.